tests/assets/setup_signing_service.py

```diff
@@ -1,46 +1,54 @@
 #!/usr/bin/env python3
```

Comment:
FYI, yes we have a manage command to setup signing services, but this script still hangs around for the versions of pulpcore that predate it (< 3.13).
Actually we don't test those anymore...